submitter/sqs.py

```diff
@@ -33,7 +33,7 @@ def message_loop(queue, wait, visibility=30):
 
 def process(msgs):
     if config.SKIP_PROCESSING != "true":
-        client = DSpaceClient(config.DSPACE_API_URL)
+        client = DSpaceClient(config.DSPACE_API_URL, timeout=config.DSPACE_TIMEOUT)
```

Comment:
@hakbailey The change looks good to me, but is there an exception raised from Dspace (client) and caught (for logging, e.g.,) if there's a login timeout?

Reply:
The DSpace client library does throw an appropriate exception in this case:
https://github.com/MITLibraries/dspace-python-client/pull/14/files

Making the exceptions more visible by implementing Sentry is part of a separate ticket in this current Sprint:
https://mitlibraries.atlassian.net/browse/ETD-443

Reply:
Yep, what Jeremy said.